.travis.yml

```
@@ -9,17 +9,19 @@ notifications:
   email: false
 
 env:
-  - GO111MODULE=on
+  - GO111MODULE=on GOTIFY_VERSIONS="master"
 
 before_install:
   - make download-tools
-  - go get
 
 script:
-  - make check
+  - go get
```

```suggestion
  - go get -d
```

We don't need to build the binary here.

---

I think for non release builds the program should be built to ensure that it compiles right?

---

Then, I would recommend using `go test ./...`, this will return 0 if the program does not have test files and compiles successfully.

If you really just want to build it, I think `go build` is better than `go get` since it does not output an executable to `$GOPATH/bin`

---

Also, you can include a test that tests the plugin against the API schema:

```golang
func TestAPICompatibility(t *testing.T) {
    assert.Implements(t, (*plugin.Plugin)(nil), new(MyPlugin))
    // Add other interfaces you intend to implement here
}
```

---

Good idea, I'll add the test and the commands you proposed.